addons/website/models/website.py

```diff
@@ -268,7 +271,8 @@ def configurator_init(self):
         r['logo'] = False
         if company_id.logo and company_id.logo != company_id._get_logo():
             r['logo'] = company_id.logo.decode('utf-8')
-        r['industries'] = self._website_api_rpc('/api/website/1/configurator/industries', {'lang': self.env.user.lang})['industries']
+        result = self._website_api_rpc('/api/website/1/configurator/industries', {'lang': self.env.user.lang})
+        r['industries'] = result and result['industries'] or False
```

Comment:
is it really intentionnal the or False ? 

Except in case result is [] it is useless

x = False and 'true' = False

Reply:
Yes it was only in the case result['industries'] is equal to [] which should never happend. So ok to remove this or